content/en/blog/2021/non-code-contribution.md

```diff
@@ -0,0 +1,48 @@
+
+To be honest, most contribution requires nothing more than Markdown and creating a pull request on GitHub. A lot of the work done in the community is in Markdown and YAML files. None of these technical skills are immediately intuitive, but it’s a lot less learning than I find most contributors know they need.
+
+My work now is focusing more on contributing to our [Contributor Marketing](https://github.com/kubernetes/community/tree/master/communication/marketing-team) subproject. The Contributor Marketing sub project's purpose is to get the word out about how to get involved. Writing this article was an output of one of those meetings. Our community does amazing things every day.
+
+None of those things required me to write code. Non-code contributors are an essential part of our community.
+
+# Hop into the Kubernetes community
+
+To join the Kubernetes community, I recommend finding a SIG where your skills align, doing things you want to do in your off time. There's a list of [Kubernetes SIGs](https://github.com/kubernetes/community/blob/master/sig-list.md); it’s had over 200 contributors over the years. If you’re not sure where to go, start with [Contributor Experience](https://github.com/kubernetes/community/tree/master/sig-contributor-experience). We’re here to help.
+
+Often, all you have to do to get started as a contributor is show up to a SIG meeting and tell folks how you want to help. Folks will be more than happy to move whatever hurdles are in your way to help you be productive. There will be some hurdles, of course--Kubernetes is a massive project--but we are all in it together.
+
+Fresh perspectives are essential to healthy projects. We need developers of all kinds to build Kubernetes and its toolchain, which is well-known. What’s lesser known is just how crucial non-code contributions are to the project’s health. If you’re interested in contributing to Kubernetes and aren’t sure how or where to get involved, feel free to join one of our [mentoring sessions](https://github.com/kubernetes/community/tree/master/mentoring). Or, if it’s easier, reach out to me directly; we’ll figure out where you belong.
```

Comment:
We don't have any active mentoring sessions at the moment. MoC hasn't had a session since Q3 last year. I would opt to just point them towards contribex in slack.

Reply:
Per contribex meeting today, MoC will be kicking back up again \o/ cc @parispittman